integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -383,6 +383,7 @@ def test_mod_pmod_by_zero_not_ansi(data_gen):
 @pytest.mark.parametrize('rhs', [byte_gen, short_gen, int_gen, long_gen, DecimalGen(6, 3),
     DecimalGen(10, -2), DecimalGen(15, 3), DecimalGen(30, 12), DecimalGen(3, -3),
     DecimalGen(27, 7), DecimalGen(20, -3)], ids=idfn)
+@pytest.mark.skipif(is_databricks113_or_later(), reason='https://github.com/NVIDIA/spark-rapids/issues/7595')
```

Comment:
It would be great if we could at least have a test that verifies that we fall back to the CPU on decimal values for this version of spark.

Even better if we check that it still works for other mixed data types, but I think that is minor if we have mod in general covered for the other non-mixed types.